twindle-cli/twitter/utils/process-tweet-entities.js

```diff
@@ -131,6 +134,8 @@ function renderOutsiderLinks(tweetObj) {
  */
 function fixUserDescription(tweets) {
   // console.log(tweets.common.user.entities);
+  if(!tweets.common.user.entities) return tweets;
```

Comment:
Also, this might fail, if either `tweets` or `tweets.common` or `tweets.common.user` is null or undefined, in rare case scenarios.

Reply:
Same scenario as above, tweets is well defined and common and user are attributes well defined. So this will most likely not happen. But if you have better ideas let me know